workflow/rules/functions.smk

```diff
@@ -17,7 +17,13 @@ def _get_container_metadata(refcon_name=None):
     else:
         refcon_md = [(key.split('_')[-1], metadata) for key, metadata in config.items() if key.startswith('metadata_')]
     if not refcon_md:
-        raise ValueError('No "metadata_" keys found in config. Did you forget to load a reference container config?')
+        base_image_name = [(key.split('_')[-1], metadata) for key, metadata in config.items() if key.startswith('metadata_')]
+        base_image_name = [(base_image, metadata) for base_image, metadata in base_image_name if base_image in select_names]
+        base_container_path = pl.Path(f'container/{base_image_name}.sif')
```

Comment:
`base_image_name` is a list here; that f-string formatting won't work as far as I can tell